docker/Makefile

```diff
@@ -417,28 +423,28 @@ else
 	@$(MAKE) -f $(THIS_FILE) migrate_only
 endif
 
-start: build postgres 
-	@echo "==> Starting blockscout"
-	@docker run --rm --name $(BS_CONTAINER_NAME) \
-					$(BLOCKSCOUT_CONTAINER_PARAMS) \
-					-p 4000:4000 \
-					$(DOCKER_IMAGE) /bin/sh -c "mix do ecto.create, ecto.migrate; mix phx.server"
-
-run: start
-
 push:
 	@echo "==> Pushing image to docker hub"
 	@docker image tag ${DOCKER_IMAGE} hermeznetwork/hermez-node-blockscout:latest
 	@docker push hermeznetwork/hermez-node-blockscout:latest
 
-start-watch:
+start:
 	@docker-compose -p "$(DOCKER_COMPOSE_PROJECT_NAME)" -f docker-compose.yml up -d
 
-stop-watch:
+stop:
 	@docker-compose -p "$(DOCKER_COMPOSE_PROJECT_NAME)" -f docker-compose.yml down --remove-orphans
 
-.PHONY: build \
+watch:
+	@docker-compose -p "$(DOCKER_COMPOSE_PROJECT_NAME)" -f docker-compose.yml up
+
+.PHONY: build-dev \
+		build-prod \
```

Comment:
This identation is wrong, all the lines should be at the same level (nothing important, but it's more readable)

Reply:
![image](https://user-images.githubusercontent.com/15896177/154984360-889dd973-a3fc-4945-8095-00831c26d9e4.png)

This is how VSCode shows them. Looks like there's an inconsistency in how GitHub and VSCode render whitespaces.

Reply:
I've changed my VScode tab to 2 spaces and committed a re-aligned version. It still does not look aligned in Github:

![image](https://user-images.githubusercontent.com/15896177/154989422-cb162f19-b053-4997-b299-8eb0e063da1b.png)

Can I change the file to spaces?